letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -367,6 +367,37 @@ BootstrapRpmCommon() {
   fi
 }
 
+BootstrapAlpineCommon() {
+  # Tested with:
+  #   - Alpine 3.4
+
+  $SUDO apk update || echo apk update hit problems but continuing anyway...
```

Comment:
Agreed, was following the pattern of the others, plus `--no-cache` was introduced in `alpine:3.3` according too https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md. I plan on keeping this implementation for the time being to allow wider support of alpine versions. I'll add a comment into the code so that others are aware of a future optimization.